.github/workflows/update-github-pages.yaml

```
@@ -0,0 +1,24 @@
+name: Deploy github pages
+on:
+  workflow_dispatch:
+jobs:
+  update-gh-pages:
+    name: Update GitHub pages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
```

La oss prøve node 18, tror det skal gå fint
```suggestion
          node-version: '18'
```